src/pages/login.tsx

```diff
@@ -1,14 +1,16 @@
-import React from 'react'
+import React, { useEffect } from 'react'
```

Comment:
Does this login page really need to exist? A redirect doesn't work?

```ts
createRedirect({
  fromPath: '/login',
  toPath: loginUrl,
  statusCode: 301, // can try remove this status code and set `redirectInBrowser: true`
})
```

Reply:
It could work but adds more complexity to the solution since it requires a lot more `createRedirects` as can be seen in [this commit](https://github.com/vtex-sites/base.store/pull/232/commits/c786e5323d6ea171e6e58ea291a01df6106d3e75) and is really tied to Gatsby. Besides that, something else is missing, maybe adding some header to the redirect requests could solve - you can see in [this preview](https://sfj-c786e53--base.preview.vtex.app/) that the login is throwing an error.
Although the redirect solution that changes the URL is slower ([this preview](https://sfj-25367af--base.preview.vtex.app/)), I'm leaning towards preferring it to the `createRedirect` one 🤔

Reply:
The advantage that using this approach with `useEffect` is the fact that we can customize the page with loading for example, but I don't know if this is enough for us to adopt. The second approach is more comfortable to me because we centralize the rules of redirects, but you had a good point when you said about the is tied to Gatsby, however, the gatsby will create this redirect on Nginx in our platform or in another thing in another platform. So, I'm leaning towards using the second approach for reason that we can separate the responsibility and the loading is not enough to keep the first approach.

Make sense for you @lariciamota ?

Reply:
It does! I'll continue working on this approach then so it can be a viable solution for us